docs/linear_system_solvers.rst

```diff
@@ -0,0 +1,61 @@
+Linear System Solvers
+=====================
+
+This section is concerned with solving problems of the form
+
+.. math::
+
+    Ax = b
+
+with unknown :math:`x` for a linear operator :math:`A` and vector :math:`b`.
+
+
+Iterative Refinement
+--------------------
+
```

Comment:
Some brief explanation of iterative refinement + link to wikipedia would be great.